pgcli/main.py

```diff
@@ -27,23 +26,44 @@
 from .config import write_default_config, load_config
 from .key_bindings import pgcli_bindings
 
+from urlparse import urlparse
+from getpass import getuser
+from psycopg2 import OperationalError
+
 _logger = logging.getLogger(__name__)
 
 @click.command()
-@click.option('-h', '--host', default='', help='Host address of the '
-        'postgres database.')
+@click.option('-h', '--host', default='localhost', envvar='PGHOST',
```

Comment:
I originally had the default for host set to `localhost` but this was broken on IPv6 machines since localhost points to ::1 and people don't generally run their postgres instance on ::1 but instead using a unix socket. 

So I changed it to an empty string since psycopg2 will do the right thing if you pass in an empty string. So I'd recommend changing this to an empty string.